src/routes/Dashboard/containers/DashboardContainer.js

```diff
@@ -7,6 +7,7 @@ const mapActionCreators = {
 const mapStateToProps = (state) => ({
   demoName: state.demoSession.name || 'loading...',
   dbdata: state.dashboard,
```

Comment:
agree - data format is likely to changes in the coming weeks. Will need to revisit this and clean up/validate.